packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -109,7 +113,7 @@ function referrerId(
   }
 
   if (!disablePersistance) {
```

Comment:
We probably don't need this check anymore since the universal storage handles this for us now right?